syntax = "proto3";

message QueryIdentifier {
  string query_id = 1;
}